linera-base/src/crypto/secp256k1.rs

```diff
@@ -349,4 +417,58 @@ mod tests {
         let key_out: Secp256k1SecretKey = serde_json::from_str(&s).unwrap();
         assert_eq!(key_out, key_in);
     }
+
+    #[test]
+    fn test_signature_serialization() {
+        use crate::crypto::{
+            secp256k1::{Secp256k1KeyPair, Secp256k1Signature},
+            TestString,
+        };
+        let keypair = Secp256k1KeyPair::generate();
+        let sig = Secp256k1Signature::new(&TestString("hello".into()), &keypair.secret_key);
+        let s = serde_json::to_string(&sig).unwrap();
+        let sig2: Secp256k1Signature = serde_json::from_str(&s).unwrap();
+        assert_eq!(sig, sig2);
+
+        let s = bcs::to_bytes(&sig).unwrap();
+        let sig2: Secp256k1Signature = bcs::from_bytes(&s).unwrap();
+        assert_eq!(sig, sig2);
+    }
+
+    #[test]
+    fn public_key_from_str() {
+        use std::str::FromStr;
+
+        use crate::crypto::secp256k1::Secp256k1PublicKey;
+        let key = Secp256k1PublicKey::test_key(0);
+        let s = key.to_string();
+        let key2 = Secp256k1PublicKey::from_str(s.as_str()).unwrap();
+        assert_eq!(key, key2);
+    }
+
+    #[test]
+    fn bytes_repr_compact_public_key() {
+        use crate::crypto::secp256k1::Secp256k1PublicKey;
+        let key_in: Secp256k1PublicKey = Secp256k1PublicKey::test_key(0);
+        let bytes = key_in.as_bytes();
+        assert!(
+            bytes.len() == 33,
```

Comment:
```suggestion
            bytes.len() == SECP256K1_PUBLIC_KEY_SIZE,
```

Reply:
Done in [Apply review suggestions.](https://github.com/linera-io/linera-protocol/pull/3407/commits/8f530c26831f602c5a89abc74abc7445a02389d6).